### ISSUE TITLE
Uncaught Error: VM Exception while processing transaction: revert

### ISSUE THREAD

**Issue by huybekool** [MyWishPlatform/airdrop-contract#1]
When  I deploy contract and test project then error , i don't know why error that @ephdtrg . Please help me

![newerror](https://user-images.githubusercontent.com/40353839/41606453-ec5d000e-740d-11e8-8c78-a6a442b181fd.png)

**Comment by ephdtrg** [comment]
I see that last of your addresses is not from provided testing CSV file, did you attached your own? 
If transaction reverted then token get out of funds, currently amount of tokens in tests is designed only for test.csv in repo

**Comment by huybekool** [comment]
Yes , I have attached Token I own and error that . If I want to use this project transfer token then how did I do  ?

**Comment by ValeryDubrava** [comment]
@huybekool 
There is a Java application to make airdrop using this contract. Please, contact with us to get it.

**Comment by ephdtrg** [comment]
@huybekool your issue fixed in 4c187278a114d1d2ea24304340a0cbb2dcaca21f, please check it.

**Comment by huybekool** [comment]
@ephdtrg many thanks  . I can test Token I own , It pass test but I have seen Token don't transfer , It just check information Address and Amout token . How did I can transfer token to Mutliaddress in a transactions as this contract You test . Example : https://ropsten.etherscan.io/tx/0x5ba89cb2f617339c1514d3f9dbb49a7ff201a3b0eceb048358ec6de7ee64c89c

**Comment by huybekool** [comment]
@ValeryDubrava how did I can contact with you to get it ? Can I contact with you via facebook ?

**Comment by ephdtrg** [comment]
@huybekool 
These tests provided to show that implemented functions work correctly, they use clear separate environment, and in default truffle configuration contract tested locally on your computer. And you testing transfer to addresses, specified in CSV file, not your token.
